backend/games/wordGuesser/ClassicWordle.py

```diff
@@ -0,0 +1,33 @@
+from typing import List
+
+
+class ClassicWordle:
+    # maximum number of guesses
+    MAX_GUESSES = 6
+    # represents the state of the game, index 0 will hold the "goal word"
+    gameBoard = List[str]
+
+    # clears any previous activity and randomizes a word to initialize the list
+    def startGame(self) -> List[str]:
+        pass
+
+    # clears all previous progress and calls checkWin() to determine final results
```

Comment:
Please change these comments to block comments

Reply:
All comments have been filtered and cleaned